### PR TITLE
finished code challenge

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/main/java/code401/challenges/tree/BinaryTree.java [hunk 1]
```diff
@@ -135,6 +135,44 @@ else if (currentLevel > 1)
         }
     }
 
+    public Integer find_maximum_value(Node<Integer> root){
+        try{
+            if(root == null)
+                return Integer.MIN_VALUE;
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/tree/BinaryTree.java:141]: Multi-line `if` statement with curly brackets is risky! Easy to make errors!
